index.php

```diff
@@ -52,9 +52,15 @@
     $totalusage = $reportconfig->filessize;
     $totaldate = date("Y-m-d H:i", $reportconfig->filessizeupdated);
 } else {
+    // Check if the path ends with a "/" otherwise an exception will be thrown
```

Comment:
Check if the "/" already exists before appending another onto it.

Reply:
Thanks for the feedback. Have made the change.